packages/cxl-lumo-styles/scss/themes/cxl-tabs-slider.scss

```diff
@@ -134,7 +135,7 @@
     margin-inline-start: calc(var(--lumo-space-m) - var(--lumo-space-xs));
 
     @media #{mq.$small} {
-      margin-inline-start: calc(var(--lumo-space-l) - var(--lumo-space-xs));
+      // margin-inline-start: calc(var(--lumo-space-l) - var(--lumo-space-xs));
```

Comment:
@freudFlintstone please remember about not leaving commented out code

Reply:
Sorry about that, solved